src/main/java/com/autotune/analyzer/deployment/InitializeDeployment.java

```diff
@@ -48,6 +46,7 @@ public static void setup_deployment_info() throws Exception, K8sTypeNotSupported
 		AutotuneDeploymentInfo.setAuthToken(auth_token);
 		AutotuneDeploymentInfo.setMonitoringAgentService(monitoring_agent_service);
 		AutotuneDeploymentInfo.setLoggingLevel(logging_level);
+		AutotuneDeploymentInfo.setRootloggingLevel(root_logging_level);
```

Comment:
Change Rootlogging to RootLogging and rootloggingLevel to rootLoggingLevel - in all places applicable